MapboxDirections/MBRouteOptions.swift

```diff
@@ -211,6 +211,10 @@ open class RouteOptions: NSObject {
      */
     open var routeShapeResolution = RouteShapeResolution.low
     
+    open var segmentAttributes: [Attribute] = []
```

Comment:
Now that Attribute is an OptionSet, we no longer need an array of Attributes; a single Attribute can store multiple options.